Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apt install -y curl && curl -fsSL https://deb.nodesource.com/setup_lts.x | b
 
 WORKDIR /app
 
-COPY . .
+#COPY . .
```

Comment:
@pjotrligthart I suggest leaving in the COPY in the dockerfile. This way if someone forgets (a.k.a does not read docs) to mount, the image still works as expected. Docker will still mount regardless of using COPY or not.

Reply:
Taking a look at this now, thanks for the input!

Reply:
@ToniSkulj I've re-enabled the COPY command, please test/review as needed